content/doc/book/system-administration/reverse-proxy-configuration-with-jenkins/reverse-proxy-configuration-lighttpd.adoc

```diff
@@ -0,0 +1,103 @@
+---
+layout: subsection
+---
+
+ifdef::backend-html5[]
+ifndef::env-github[:imagesdir: ../../../resources/managing]
+:notitle:
+:description:
+:author:
+:email: jenkinsci-users@googlegroups.com
+:sectanchors:
+:toc: left
+endif::[]
+
+[[running-jenkins-behind-lighttpd]]
+= Reverse proxy - Lighttpd
+
+In situations where you have existing web sites on your server, you may
+find it useful to run Jenkins (or the servlet container that Jenkins
+runs in) behind https://www.lighttpd.net/[Lighttpd], so that you can bind Jenkins
+to the part of a bigger website that you may have or just centralize the TLS
+termination in a single place.
+This section discusses some of the approaches for doing this.
+
+The https://redmine.lighttpd.net/projects/lighttpd/wiki/Mod_proxy[mod_proxy] works
+by making Lighttpd perform as a "reverse proxy" — when a request arrives for
+certain URLs, Lighttpd becomes a proxy and forwards that request to
+Jenkins, then forwards the response from Jenkins back to the client. 
```

Comment:
```suggestion
The link:https://redmine.lighttpd.net/projects/lighttpd/wiki/Mod_proxy[mod_proxy] works
by making Lighttpd perform as a "reverse proxy".
This means when a request arrives for certain URLs, Lighttpd becomes a proxy and forwards that request to Jenkins, then forwards the response from Jenkins back to the client. 
```
just trying to break up the sentences a bit/format accordingly, again, if this takes away from the message please disregard